blog/content/post/2019-10-24-meta-pcengines-xen.md

```diff
@@ -0,0 +1,101 @@
+---
+title: 'XEN on the PC Engines apu2 - why you want to use it'
+abstract: 'Xen Project creates a software system that allows the execution of multiple virtual guest operating systems simultaneously on a single physical machine. In this case, it is a PC Engines apu2 platform. '
+cover: /covers/3mdeb-logo_simple-color.png
+author: norbert.kaminski
+layout: post
+published: false
+date: 2019-10-24
+archives: "2019"
+
+tags:
+  - XEN
+  - meta-pcengines
+  - apu2
+  - PC engines
+categories:
+  - OS Dev
+  - Security
+  - Manufacturing
+
+---
+# What is the Xen Project?
+
+Xen Project creates a software system that allows the execution of multiple virtual guest operating systems simultaneously on a single physical machine. It is valuable when you want to create a complex security software system. None of the guest operating systems have direct access to the host hardware.
+
+A Xen hypervisor manages the virtual machine guests known as domains. It is controlled by a special domain called domain0 (dom0). Only the hypervisor has access to the physical outputs of the platform. Depending on the guest type (HVM or PV) hypervisor allows unprivileged domains (domUs) to use virtual hardware devices or passes the links to the physical hardware.
```

Comment:
@Asiderr Please wrap lines at 80chars. Whole paragraph in a single line is not easy to review.